### PR TITLE
Leo/design update

### PR DESCRIPTION
**Pull request by leonardovenoso**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Design refactoring according to Figma

#### Which issue(s) does this PR fixes?:

Refactoring home page transactions and blocks sections according to Figma design
- Change icon positioning only for liquidity pools
- Extracted duplicated code under Collapsible tag to a function
- Removed VIEW collapsible button as requested in the design
- Updated the cards layouts for mobile and desktop
- Updated integration tests for mobile and desktop
- Fixed test: search bar should be visible for mobile
- Amount text is inline with numeric amount and token
- Reorganization of transaction information for mobile
- Removed search bar from header only on HomePage.

#### Sample Links & Screenshots:

Link: 
https://deploy-preview-725--defi-scan.netlify.app/

<details>
<summary>Desktop Screenshot</summary>

Screenshot 2021-12-13 at 10 03 52 AM

</details>
<details>
<summary>Mobile Screenshot</summary>

Screenshot 2021-12-13 at 10 04 09 AM

</details>

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on multiple web browsers
- [x] Tested responsiveness (e.g, iPhone, iPad, Desktop)
- [x] No console errors
- [ ] Unit tests*
- [x] Added e2e tests*

### CODE REVIEW

File: src/components/index/TransactionsList.tsx [hunk 1]
```diff
@@ -2,135 +2,89 @@ import { Transaction } from '@defichain/whale-api-client/dist/api/transactions'
...
+          </div>
+          <div className='flex-none w-9' />
+          <div className='flex-grow w-10'>
+            <div className='hidden md:block text-sm'>
+              <span className='text-gray-400'>Amount&nbsp;</span>
+              <span>{`${new BigNumber(props.totalVoutValue).toFixed(8)} DFI`}</span>
+            </div>
+            <div className='md:hidden text-center text-xs text-gray-400'>
+              <span>{props.age}</span>
+            </div>
```
Comment (joeldavidw) [inline src/components/index/TransactionsList.tsx:72]: Try to get the values more aligned to the right. I think you could reuse what was done before just make the texts closer. Similarly for Blocks too.

image

**Review by joeldavidw — CHANGES_REQUESTED** [review]

File: src/components/index/TransactionsList.tsx (hunk 1 above)
Comment (leonardovenoso) [reply src/components/index/TransactionsList.tsx:72]: Ok, noted

File: src/components/index/TransactionsList.tsx [hunk 2]
```diff
@@ -2,135 +2,89 @@ import { Transaction } from '@defichain/whale-api-client/dist/api/transactions'
...
 
-function DesktopTransactionDetails (props: { totalVoutValue: string }): JSX.Element {
+function TransactionDetails (props: { txid: string, age: string, totalVoutValue: string }): JSX.Element {
   return (
-    <div className='hidden md:block w-1/2 lg:w-3/5 xl:w-2/5'>
-      <div className='w-full flex'>
-        <div className='w-1/2 text-right text-sm text-gray-500 lg:mr-2 xl:mr-0'>
-          Amount
+    <div className='flex p-4 rounded border border-gray-200 my-1.5 hover:shadow-md'>
+      <div className='flex-none w-11/12 lg:w-18/19'>
```
[the remark this answers is not in the document]
Comment (leonardovenoso) [reply src/components/index/TransactionsList.tsx:51]: Ok, noted

File: src/components/index/TransactionsList.tsx (hunk 1 above)
Comment (leonardovenoso) [reply src/components/index/TransactionsList.tsx:72]: @joeldavidw I tried to reuse that code but I saw code repeated and hard to reason about, specially in the layout. I suggest to use the new version.

Comment (leonardovenoso) [reply src/components/index/TransactionsList.tsx:72]: May I know the resolution you used?

Comment (leonardovenoso) [reply src/components/index/TransactionsList.tsx:72]: Is it OK for you like this alignment?

Screenshot 2021-12-15 at 2 57 13 AM

Comment (leonardovenoso) [reply src/components/index/TransactionsList.tsx:72]: Is it OK for you this alignment?

Screenshot 2021-12-15 at 7 07 14 PM

**Comment by leonardovenoso** [comment]
As discussed attached align amounts to the right

Screenshot 2021-12-15 at 7 14 48 PM

**Pull request closed without merging**